### DISCUSSION TITLE
I would like to build v2ray plug in for iOS

### DISCUSSION THREAD

**Discussion by mahirozdin** [v2fly/v2ray-core#2945] [Q&A]
I couldn't find how to integrate v2ray for ios I have a vpn app for android and I'm able to connect as v2ray client but could not find any library or anything on ios 

Can anyone help me with that how can I create a client for ios

**Comment by ghost** [comment]
tell me your email

**Reply by hlheart** [reply]
> tell me your email

2858796570@qq.com 非常感谢🙏，付费
